lib/orwell/web/router.ex

```diff
@@ -28,6 +28,8 @@ defmodule Orwell.Web.Router do
     delete "/auth", AuthController, :delete
     get "/auth/:provider", AuthController, :request
     get "/auth/:provider/callback", AuthController, :callback
+
+    get "/*path", PostController, :show
```

Comment:
Or even take a page out of the Twitter book and have it be `/p/pages/posts/path-to-post.md`